rust/src/passwd.rs

```diff
@@ -630,8 +631,13 @@ pub fn check_passwd_group_entries(
 
     // Parse entries in the upcoming commit content.
     let mut new_entities = PasswdEntries::default();
-    new_entities.add_passwd_content(rootfs.as_raw_fd(), "usr/lib/passwd")?;
-    new_entities.add_group_content(rootfs.as_raw_fd(), "usr/lib/group")?;
+    if has_usrlib_passwd(&rootfs)? {
+        new_entities.add_passwd_content(rootfs.as_raw_fd(), "usr/lib/passwd")?;
+        new_entities.add_group_content(rootfs.as_raw_fd(), "usr/lib/group")?;
+    } else {
+        new_entities.add_passwd_content(rootfs.as_raw_fd(), "usr/etc/passwd")?;
```

Comment:
Is this right though?  I thought we'd be relying on systemd-sysusers creating the users/groups on firstboot?

Reply:
Maybe should add both? Do you mean with empty `passwd` / `group` (and no `check-passwd/check-groups`)? Maybe that is the final goal.

Reply:
Right, I thought the goal was that we start with an empty `passwd` file if we were going all-in on sysusers.